.settings.xml

```diff
@@ -1,60 +1,70 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <settings>
-  <servers>
-	<server>
-	  <id>repo.spring.io</id>
-	  <username>${env.CI_DEPLOY_USERNAME}</username>
-	  <password>${env.CI_DEPLOY_PASSWORD}</password>
-	</server>
-  </servers>
-  <profiles>
-	<profile>
-	  <id>spring</id>
-	  <activation><activeByDefault>true</activeByDefault></activation>
-	  <repositories>
-		<repository>
-			<id>spring-snapshots</id>
-			<name>Spring Snapshots</name>
-			<url>http://repo.spring.io/libs-snapshot-local</url>
-			<snapshots>
-				<enabled>true</enabled>
-			</snapshots>
-		</repository>
-		<repository>
-			<id>spring-milestones</id>
-			<name>Spring Milestones</name>
-			<url>http://repo.spring.io/libs-milestone-local</url>
-			<snapshots>
-				<enabled>false</enabled>
-			</snapshots>
-		</repository>
-		<repository>
-			<id>spring-releases</id>
-			<name>Spring Releases</name>
-			<url>http://repo.spring.io/release</url>
-			<snapshots>
-				<enabled>false</enabled>
-			</snapshots>
-		</repository>
-	  </repositories>
-	  <pluginRepositories>
-		<pluginRepository>
-			<id>spring-snapshots</id>
-			<name>Spring Snapshots</name>
-			<url>http://repo.spring.io/libs-snapshot-local</url>
-			<snapshots>
-				<enabled>true</enabled>
-			</snapshots>
-		</pluginRepository>
-		<pluginRepository>
-			<id>spring-milestones</id>
-			<name>Spring Milestones</name>
-			<url>http://repo.spring.io/libs-milestone-local</url>
-			<snapshots>
-				<enabled>false</enabled>
-			</snapshots>
-		</pluginRepository>
-	  </pluginRepositories>
-    </profile>
-  </profiles>
+	<servers>
+		<server>
+			<id>repo.spring.io</id>
+			<username>${env.CI_DEPLOY_USERNAME}</username>
+			<password>${env.CI_DEPLOY_PASSWORD}</password>
+		</server>
+	</servers>
+	<profiles>
+		<profile>
+			<id>spring</id>
```

Comment:
Yeah, I don't think the `servers` section is needed. I see that the CI build plan doesn't use it. Addressed the above.